app/iCubTest/test.ini

```diff
@@ -0,0 +1,14 @@
+[REFERENCES]
+user "alessandro scalzo"
+comment "this is a wild bunch of tests"
+outfile iCubTestReport.xml
+
+[ENVIRONMENT]
+robotname icub
+
+[TESTS]
+TestCamera right_camera.ini
+TestCamera left_camera.ini
+TestMotors test_left_arm.ini
+TestMotors test_right_arm.ini
+
```

Comment:
Some sensors are available on only some iCubs. Should do we move at least the test.ini file in robots specific directories?